helix-contract/contracts/mapping-token/v2/native-parachain-protocol/NativeParachainBacking.sol

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import "@zeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "@darwinia/contracts-periphery/contracts/s2s/types/PalletHelixBridge.sol";
+import "../Backing.sol";
+import "../../interfaces/IHelix2ParaMessageEndpoint.sol";
+import "../../../utils/BoundedVec.sol";
+
+contract NativeParachainBacking is Backing {
+    struct LockedInfo {
+        address payable sender;
+        uint256 amount;
+    }
+    uint256 public helixFee;
+    uint256 public kPrunSize;
+
+    bytes2 public remoteIssuingIndex;
+    bytes2 public remoteHandleIssuingForFailureIndex;
+
+    // (nonce => lockedInfo)
+    mapping(uint64 => LockedInfo) public lockedMessages;
+    BoundedVec.Uint64Vec acceptedNonces;
```

Comment:
What is the purpose of the `acceptedNonces` design, I need more information to review.

Reply:
please ref to https://www.notion.so/itering/smart-chain-parachain-1597d985536f4ea9925e0bee0889c5f0